### PR TITLE
feat: enhance accessibilty and seo for hero section

### LINKED ISSUE #530: Feature: add accessibility and seo to the hero section
### Suggested solution

in the hero.tsx i would like to change the regular div with semantic elements and add labels for enhancing and helping screen readers

### PR DESCRIPTION
**Pull request by sohaibchebah**
## Description

This PR adds more accessibility to the hero section and also seo bost becuase search engine crawlers understand more the semantic html  

## Related Tickets & Documents

Closes #530 

## Mobile & Desktop Screenshots/Recordings

N/A

## Steps to QA
N/A

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

### CODE REVIEW

**Comment by nickytonline** [comment]
Hi @sohaibchebah! Please read the contributing guidelines above.

File: src/components/Hero.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
 const Hero = () => (
-  <section aria-labelledby="hero-heading" className="flex flex-col py-24 items-center mx-2.5">
+  <section
+    aria-labelledby="hero-heading"
```
Comment (nickytonline) [inline src/components/Hero.tsx:3]: refactor (blocking): assistive technologies know how to pick up the header within the section, so no need for aria.

```suggestion
  <section
```

File: src/components/Hero.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+    className="flex flex-col py-24 items-center mx-2.5">
     <div>
       <h1
-        id="hero-heading"
         className="font-Lexend text-4xl md:text-5xl text-center text-lightSlate leading-tight tracking-tight"
+        id="hero-heading"
```
Comment (nickytonline) [inline src/components/Hero.tsx:8]: refactor (blocking): not required base on comment above.
```suggestion
```

**Review by nickytonline — CHANGES_REQUESTED** [review]
Thanks for making things more accessible @sohaibchebah!

File: src/components/Hero.tsx (hunk 1 above)
Comment (sohaibchebah) [reply src/components/Hero.tsx:3]: thanks a lot for the information 😊

**Comment by nickytonline** [comment]
This repo is being archived.

**Pull request closed without merging**